src/Mechanical/TranslationalPosition/sensors.jl

```diff
@@ -0,0 +1,81 @@
+"""
+    ForceSensor(; name)
+
+Linear 1D force input sensor.
+
+# Connectors:
+
+- `flange_a`: 1-dim. translational flange
+- `flange_b`: 1-dim. translational flange
+- `output`: real output
+"""
+@mtkmodel ForceSensor begin
+    @components begin
+        flange_a = Flange()
+        flange_b = Flange()
+        output = RealOutput()
+    end
+
+    @equations begin
+        flange_a.s ~ flange_b.s
+        flange_a.f + flange_b.f ~ 0.0
+        output.u ~ flange_a.f
+    end
+end
+
+"""
+    PositionSensor(; s = 0, name)
+
+Linear 1D position input sensor.
```

Comment:
what does "position input" mean?
```suggestion
Linear 1D position sensor.
```

Reply:
I had actually copied it from Translational. I have fixed in both in [35a3947](https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/303/commits/35a3947d4f0b27ee155716336bc681c9ff55d06f)